saleor/webhook/transport/asynchronous/transport.py

```diff
@@ -164,6 +164,17 @@ def create_event_delivery_list_for_webhooks(
     return event_deliveries
 
 
+def get_queue_name_for_webhook(webhook, default_queue):
+    scheme = urlparse(webhook.target_url).scheme.lower()
+    if scheme == WebhookSchemes.AWS_SQS:
+        queue_name = settings.WEBHOOK_SQS_CELERY_QUEUE_NAME
+    elif scheme == WebhookSchemes.GOOGLE_CLOUD_PUBSUB:
+        queue_name = settings.WEBHOOK_PUBSUB_CELERY_QUEUE_NAME
+    else:
+        queue_name = default_queue
+    return queue_name
```

Comment:
This is pure nitpicking, but I don't know why we need to assign to `queue_name` and then return it. Why not:
```suggestion
def get_queue_name_for_webhook(webhook, default_queue):
    scheme = urlparse(webhook.target_url).scheme.lower()
    if scheme == WebhookSchemes.AWS_SQS:
       return settings.WEBHOOK_SQS_CELERY_QUEUE_NAME
    if scheme == WebhookSchemes.GOOGLE_CLOUD_PUBSUB:
        return settings.WEBHOOK_PUBSUB_CELERY_QUEUE_NAME
    return default_queue
```

Reply:
or even abandon if-else approach
```suggestion
def get_queue_name_for_webhook(webhook, default_queue):
    return {
        WebhookSchemes.AWS_SQS: settings.WEBHOOK_SQS_CELERY_QUEUE_NAME,
        WebhookSchemes.GOOGLE_CLOUD_PUBSUB: settings.WEBHOOK_PUBSUB_CELERY_QUEUE_NAME,
    }.get(
        urlparse(webhook.target_url).scheme.lower(),
        default_queue,
    )
```